reference/ros-api.md

```diff
@@ -347,6 +347,7 @@ for more information about the value of `robot_unique_name`).
 | [/supervisor/node/get\_id](supervisor.md#wb_supervisor_node_get_from_def)                                          | service       | webots\_ros::node\_get\_id                           | uint64 node<br/>---<br/>int32 id                                                                                                                                                                                                                                  |
 | [/supervisor/node/get\_type](supervisor.md#wb_supervisor_node_get_type)                                            | service       | webots\_ros::node\_get\_type                         | uint64 node<br/>---<br/>int32 type                                                                                                                                                                                                                                |
 | [/supervisor/node/get\_type\_name](supervisor.md#wb_supervisor_node_get_type)                                      | service       | webots\_ros::node\_get\_type\_name                   | uint64 node<br/>---<br/>string name                                                                                                                                                                                                                               |
+| [/supervisor/node/get\_def](supervisor.md#wb_supervisor_node_get_def)                                              | service       | webots\_ros::node\_get\_def                          | uint64 node<br/>---<br/>string name                                                                                                                                                                                                                               |
```

Comment:
Here now you should use `webots_ros::node_get_name`, the other two `services get_type_name` and `get_base_type_name` have to be updated too.